pkg/codec/cloudevents/json/json.go

```diff
@@ -23,7 +23,7 @@ import (
 type (
 	Config struct {
 		SpecVersion string `mapstructure:"specversion"`
-		Source      string `mapstructre:"source"`
+		Source      string `mapstructure:"source" structs:"-"`
```

Comment:
`structs:"-"`

What is this for?

Reply:
It was for the old validator. Didn't notice that didn't get removed. It's gone now.